azure-core/dev_requirements.txt

```diff
@@ -1,3 +1,4 @@
 trio; python_version >= '3.5'
 aiohttp>=3.0; python_version >= '3.5'
-aiodns>=2.0; python_version >= '3.5'
\ No newline at end of file
+aiodns>=2.0; python_version >= '3.5'
+pylint>=2.3; python_version >= '3.5'
```

Comment:
Please also add a pylint version for Python < 3.5. I think it's like 1.8 or something.

Reply:
0.1 off. Python 2.7 support was dropped in 2.0. So we should use 1.9.x for Python 2.7.